src/platforms/ios/ioscontroller.swift

```diff
@@ -178,6 +176,8 @@ public class IOSControllerImpl : NSObject {
             configHack["isSuperDooperFeatureActive"] = isSuperDooperFeatureActive
             configHack["gleanDebugTag"] = gleanDebugTag
             configHack["installationId"] = installationId
+
+            configHack["fallbackConfig"] = fallbackConfig?.asWgQuickConfig() ?? ""
```

Comment:
Random: at this point we should consider renaming this 'configHack' to something more semantic. We are using it for so many things already.

Reply:
Good call. I've updated it to `configManipulation`. If you have an even better name, please let me know - I don't think `configManipulation` is the best one possible, but it's the best I'm coming up with right now.

Reply:
Hm, SGTM... Brainstorming: 'configExtension'? 'extraConfig'? 'customConfig'? 'mozConfig'?

Reply:
I like `customConfig; I've updated this.